src/Avalonia.Base/DebugDisplayHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Text;
+
+namespace Avalonia;
+
+internal static class DebugDisplayHelper
```

Comment:
Can you move it to Diagnostics sub-folder? Just to keep root folder from getting way too big.

Reply:
Sure!